src/lib/pubkey/pk_algs.cpp

```diff
@@ -84,7 +84,9 @@ std::unique_ptr<Public_Key>
 load_public_key(const AlgorithmIdentifier& alg_id,
                 const std::vector<uint8_t>& key_bits)
    {
-   const std::string alg_name = OIDS::lookup(alg_id.oid);
+   const std::vector<std::string> alg_info = split_on(OIDS::lookup(alg_id.oid), '/');
+   const std::string alg_name = alg_info[0];
```

Comment:
I think if the OID is unknown this can access an empty vector

Reply:
fixed. Seems like the fuzzers don't compile on Windows, so at the moment i have no way to verify local on my machine if this is causing the fuzzer crashes.